react/SearchWrapper.tsx

```diff
@@ -94,13 +94,13 @@ const SearchWrapper: FC<SearchWrapperProps> = props => {
     params,
     searchQuery,
     searchQuery: {
-      loading = true,
       data: { searchMetadata: { titleTag = '', metaTagDescription = '' } = {} } = {},
     } = {},
     children,
   } = props
   const { account, getSettings } = useRuntime()
   const settings = getSettings(APP_LOCATOR) || {}
+  const loading = searchQuery.loading
```

Comment:
I had to do this because of TS error:

![image](https://user-images.githubusercontent.com/284515/70190933-0f13f500-16d6-11ea-89ec-84966603d4d5.png)

The solution to this error caused this bug.